corehq/apps/app_manager/templates/app_manager/app_diff.html

```diff
@@ -0,0 +1,49 @@
+{% extends "style/bootstrap3/base_section.html" %}
```

Comment:
Semi-related, when you expand a long diff, the HQ footer covers the bottom of the page (download_index doesn't seem to have a problem with this).